hotel/site_sections/hotel_requests.py

```diff
@@ -31,9 +31,12 @@ def index(self, session, message='', decline=None, **params):
                 requests.nights = ','.join(map(str, c.CORE_NIGHTS))
 
         nights = []
+        2day_before = (c.EPOCH - timedelta(days=2)).strftime('%A')
         day_before = (c.EPOCH - timedelta(days=1)).strftime('%A')
         last_day = c.ESCHATON.strftime('%A').upper()
         day_after = (c.ESCHATON + timedelta(days=1)).strftime('%A')
+        nights.append([getattr(c, 2day_before.upper()), getattr(requests, 2day_before.upper()),
+                       "I'd like to help set up on " + 2day_before])
```

Comment:
Looks good overall, but the main thing that needs to change is that Python variable names cannot start with numbers.  So you'd need to name the variable something like ``two_days_before`` instead of ``2day_before`` since the latter would give an error.